{{cookiecutter.github_repository}}/provisioner/roles/postgresql/tasks/main.yml

```diff
@@ -9,6 +9,12 @@
     - postgresql-contrib
     - python-psycopg2
 
+- name: install postgis packages for Ubuntu
+  apt:
+    pkg: "postgis"
+    state: installed
+  when: pg_gis
```

Comment:
I believe this needs to be in the `common` role. Just in case you don't want to use the postgres db role in the same web instance.